templates/cluster/azure-standalone-cp/templates/k0scontrolplane.yaml

```diff
@@ -10,6 +10,22 @@ spec:
     {{- if $global.k0sURL }}
     downloadURL: "{{ $global.k0sURL }}/k0s-{{ .Values.k0s.version }}-amd64"
     {{- end }}
+    {{- $certs := dict "registry.crt" $global.registryCertSecret "k0s-url.crt" $global.k0sURLCertSecret }}
```

Comment:
Minor: Shouldn't the dict be declared only if either of the values below exists? Applicable elsewhere if that even matters (if no, then just resolve the comment)

Reply:
Yeah, it makes sense. Thanks. I'll update the PR.